src/osemosys2iamc/resultify.py

```diff
@@ -536,6 +536,16 @@ def main(config: Dict, inputs_path: str, results_path: str) -> pyam.IamDataFrame
         all_data = all_data.convert_unit("MEUR_2015/GW", to="EUR_2020/kW", factor=1.05)
         all_data = all_data.convert_unit("kt CO2/yr", to="Mt CO2/yr")
 
+        dic_country_name_variants = {
+            "Netherlands": "The Netherlands",
+            "Czechia": "Czech Republic",
+            "United Kingdom of Great Britain and Northern Ireland": "United Kingdom",
+        }
+        for c in dic_country_name_variants:
+            all_data.data["region"] = all_data.data["region"].str.replace(
+                c, dic_country_name_variants[c]
+            )
```

Comment:
pyam can handle rename/replace directly in a much more efficient manner.
```suggestion
        all_data.rename(region=dic_country_name_variants, inplace=True)
```

Note that you could also use the `inplace=True` feature in the unit-conversion just above this section...